src/index.html

```diff
@@ -7,12 +7,51 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="MacBook"
+        />
+      </div>
+      <p class="card__text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="card__kod">Product code: 195434</p>
+      <div class="card__review-wrapper">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
```

Comment:
The 'stars' block seems to be a reusable component. It's better to separate the number of active stars from the block modifier (e.g., 'stars--4'). You could have a separate class to denote active stars like 'stars__star--active' and apply it to the first four stars.